src/server/index.test.jsx

```diff
@@ -132,3 +117,47 @@ describe('Server', () => {
     });
   });
 });
+
+describe('Server HTTP Header', () => {
+  const makeRequest = async path => request(server).get(path);
+
+  it(`should not contain 'x-powered-by'`, async () => {
+    const { headers } = await makeRequest('/status');
```

Comment:
We can just make this request once at the top of the describe block, to save making multiple requests.